homeassistant/config.py

```diff
@@ -741,13 +741,13 @@ def config_without_domain(config: Dict, domain: str) -> Dict:
 
     This method is a coroutine.
     """
-    from homeassistant.scripts.check_config import check_ha_config_file
+    import homeassistant.helpers.check_config as check_config
```

Comment:
Why do we import inside the function? Can we move the import to the top of the module?

Reply:
Initially we did that because it would pull in all the script stuff, and we didn't want that. Now that it's a helper we can just import at the top.

Reply:
The helper imports config an config imports the helper. Next PR I could try change this in config&web view

Reply:
Put the import on top of this file

Reply:
Not sure I can do that, because of the circular import. 

If I move it to the top here, I have to move a whole list of imports into a function in the helper

Better to refactor/remove this method from config.py in the long run